packages/broker/src/plugins/operator/MaintainTopologyService.ts

```diff
@@ -1,49 +1,38 @@
 import { Logger, Multimap } from '@streamr/utils'
 import { MaintainTopologyHelper } from './MaintainTopologyHelper'
 import StreamrClient, { Stream, Subscription } from 'streamr-client'
-import { StreamID, StreamPartIDUtils, toStreamID } from '@streamr/protocol'
-import { SetMembershipSynchronizer } from '../storage/SetMembershipSynchronizer'
+import { StreamID, StreamPartIDUtils } from '@streamr/protocol'
 import pLimit from 'p-limit'
 
-function toStreamIDSafe(input: string): StreamID | undefined {
-    try {
-        return toStreamID(input)
-    } catch {
-        return undefined
-    }
-}
+const logger = new Logger(module)
 
 export class MaintainTopologyService {
     private readonly streamrClient: StreamrClient
     private readonly maintainTopologyHelper: MaintainTopologyHelper
     private readonly subscriptions = new Multimap<StreamID, Subscription>()
```

Comment:
Could put the `subscription` s the first field as this is the main state of the class. Others are just dependencies.

Reply:
I prefer to have fields ordered from tightest to least tight (w.r.t visibility and readonlyness), E.g.
```
private readonly
private readonly
protected readonly
protected readonly
public readonly
public readonly

private
private
protected
protected
public
public
```